### PR TITLE
[Refact] Add assertJ and bytebuddy and refactor TestExample

### PR DESCRIPTION
**Pull request by boutss**
1. convert to assertJ
2. modify TestExample for test a error case

### CODE REVIEW

**Comment by boutss** [comment]
Thanks for the review.
Ok for the check in jenkins in normal mode, that's great this !

File: src/main/java/com/efluid/tcbc/Classe.java [hunk 1]
```diff
@@ -48,8 +52,8 @@ public Jar getJar() {
     return jar;
   }
 
-  public void afficherJar() {
-    ScanneClasspath.doLog("\t- " + getJar().getNom());
+  public String getNomJar() {
```
Comment (tcollignon) [inline src/main/java/com/efluid/tcbc/Classe.java]: faudra une passe de tout remettre en anglais :)

Comment (boutss) [reply src/main/java/com/efluid/tcbc/Classe.java]: Oui je ferai un PR pour remettre in english !

Comment (boutss) [reply src/main/java/com/efluid/tcbc/Classe.java]: who merge the PR so ? Me ?

**Pull request merged**